app/main/api/handlers/newMnemonics.ts

```diff
@@ -0,0 +1,20 @@
+import { asObject } from "app/common/runtimeTypes"
+import { SubSystems } from "app/main/system"
+import { generate } from "app/main/crypto/mnemonic"
```

Comment:
Importing the entire `mnemonic` module so that we can use `generate()` as `mnemonic.generate()` is a little more clear, IMO.